src/topology/algebra/group.lean

```diff
@@ -46,11 +46,22 @@ lemma continuous.inv [topological_group α] [topological_space β] {f : β → 
   (hf : continuous f) : continuous (λx, (f x)⁻¹) :=
 continuous_inv.comp hf
 
+@[to_additive]
+lemma continuous_on_inv [topological_group α] {s : set α} : continuous_on (λx:α, x⁻¹) s :=
+continuous_inv.continuous_on
+
 @[to_additive]
 lemma continuous_on.inv [topological_group α] [topological_space β] {f : β → α} {s : set β}
   (hf : continuous_on f s) : continuous_on (λx, (f x)⁻¹) s :=
 continuous_inv.comp_continuous_on hf
 
+
+@[to_additive]
+lemma filter.tendsto_inv {α : Type*} [group α]
```

Comment:
You can just use `tendsto_inv`, without the `filter`: when you want to use the dot notation, it is important to have lemmas named `filter.tendsto.something`, so that they apply to an assumption of the form `filter.tendsto ...`. Here, it's not the case, you can't use dot notation with this one, so no need to be in the filter namespace.

Reply:
Oh I see, I didn't see it that way but that's a good decision rule